Formula/coccinelle.rb

```diff
@@ -25,7 +25,9 @@ class Coccinelle < Formula
   depends_on "hevea" => :build
   depends_on "ocaml-findlib" => :build
   depends_on "opam" => :build
+  depends_on "pkg-config" => :build
   depends_on "ocaml"
+  depends_on "pcre"
```

Comment:
Can this use `pcre2`?

Reply:
I haven't tried. I mainly based on other formula that use both ocaml and pcre:
```
haxe.rb
37:  depends_on "pcre"

comby.rb
22:  depends_on "pcre"

semgrep.rb
30:  depends_on "pcre"
```

Looking at discussion in `pcre-ocaml` https://github.com/mmottl/pcre-ocaml/issues/10 it sounds like there was no plan to support `pcre2` unless someone else contributed it.